saleor/account/models.py

```diff
@@ -6,15 +6,19 @@
     BaseUserManager,
     PermissionsMixin,
 )
+from django.contrib.postgres.fields import JSONField
 from django.db import models
 from django.db.models import Q, Value
 from django.forms.models import model_to_dict
 from django.utils import timezone
+from django.utils.timezone import now
```

Comment:
Maybe you can use `timezone.now` instead of direct `now` call, then this import will be not needed

Reply:
Django's `timezone.now` looks at django settings for proper timezone info, whether it should use UTC or the local timezone.

Reply:
Yes, I know but I meant that you have two similar imports here.
```
from django.utils import timezone
from django.utils.timezone import now
```

Reply:
Oh good catch! I may be blind! Thanks!